packages/app-runtime/package.json

```
@@ -23,11 +23,21 @@
         "cdep": "tsc && madge --circular dist",
         "lint:tsc": "tsc --noEmit && tsc -p test/tsconfig.json --noEmit",
         "test": "jest --forceExit",
-        "test:ci": "jest -i --forceExit"
+        "test:coverage": "npm run test -- --coverage=true",
+        "test:ci": "jest -i --forceExit --coverage=true"
```

```suggestion
        "test:coverage": "npm run test -- --coverage",
        "test:ci": "jest -i --forceExit --coverage"
```

same for all other occurences.